editor/editor_node.cpp

```diff
@@ -7703,7 +7795,7 @@ EditorNode::EditorNode() {
 	if (global_menu && NativeMenu::get_singleton()->has_system_menu(NativeMenu::APPLICATION_MENU_ID)) {
```

Comment:
Do we need to take into account `NativeMenu::get_singleton()->has_system_menu(NativeMenu::APPLICATION_MENU_ID)` as well?

cc @bruvzg